src/blockdata/script.rs

```diff
@@ -118,13 +118,16 @@ pub enum Error {
     EarlyEndOfScript,
     /// Tried to read an array off the stack as a number when it was more than 4 bytes
     NumericOverflow,
-    #[cfg(feature="bitcoinconsensus")]
+    #[cfg(feature = "bitcoinconsensus")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "bitcoinconsensus")))]
```

Comment:
Lol whoops, we definitely should not have feature-gated enum variants. Surprised nobody has complained about this.

Looks like Tamás and I did this [in March 2018](https://github.com/rust-bitcoin/rust-bitcoin/pull/58).

Nothing to do with this PR but we should fix it in another one.

Reply:
Yeah, for normal enums that'd be breaking because some crate could do exhaustive match on it while other crate turns on the feature. I think it'd be fine for `#[non_exhaustive]` enums.

Reply:
Opened #645 with a suggested solution.